Update active-directory-application-proxy-wildcard.md

adamfowlerit opened the pull request:
Updated the yourAADTenantId name to match in both fields, and fixed the missing .tenant part of the response.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

adamfowlerit committed:
Update active-directory-application-proxy-wildcard.md

Updated the yourAADTenantId name to match in both fields, and fixed the missing .tenant part of the response.
